platform/ui/src/views/StudyList/components/StudyListFilter.js

```diff
@@ -241,19 +143,24 @@ const StudyListFilter = ({
                         'pl-4 first:pl-12'
                       )}
                     >
-                      <FilterLabel
-                        key={name}
-                        label={displayName}
-                        isSortable={
-                          isSortable && numOfStudies <= 100 && numOfStudies > 0
-                        }
-                        isBeingSorted={sortBy === name}
-                        sortDirection={sortDirection}
-                        onLabelClick={() => handleFilterLabelClick(name)}
-                        inputType={inputType}
-                      >
-                        {renderInput(inputType, name, { selectOptions })}
-                      </FilterLabel>
+                      {FilterInputComponent && (
```

Comment:
Just mark as required?

Reply:
This is for cases like "Instances" where we don't have an input but we have the label and sorting. 
Shoul we treat that in a different way? Like creating a component for that expecifically?

Reply:
Good question. Is the "just a label" a component? Maybe I skipped over where/how you're rendering it.

Reply:
As it could have sorting on it, I have used the same stucture but I don't add an input for it.